features/step_definitions/search_bar_steps.rb

```diff
@@ -0,0 +1,28 @@
+  Given('I have searched for {string}') do |query|
```

Comment:
Please remove the redundant, unused step definitions

Reply:
This step definition is actually being used. Please refer to Scenario 3 in search_bar.feature file.

Reply:
Oh sorry, I missed that.  Could we replace the code in there with step definitions then?
```
Given(...) do |query|
  steps %(
    Given I file in "query" with #{query}
    And I click the "Search Assignment" button
  )

Reply:
Yeah, we can do that.